lib/graphql/tracing/appoptics_tracing.rb

```diff
@@ -112,6 +112,8 @@ def metadata(data, layer)
             graphql_query_string(data[key])
           when :multiplex
             graphql_multiplex(data[key])
+          when :path
+            [key, data[key].join(".")]
```

Comment:
I wonder if this ever worked on the interpreter. It was returning `[:path, []]`, and `[]` was flattened out of existence by the `.flatten(2)` call below, then `.each_slice(2)` raised an error about getting an odd-length array.